packages/consumption/test/modules/requests/itemProcessors/readAttribute/ReadAttributeRequestItemProcessor.test.ts

```diff
@@ -353,16 +353,16 @@ describe("ReadAttributeRequestItemProcessor", function () {
         });
 
         test("returns an error when the given Attribute id belongs to a peer Attribute", async function () {
-            const peer = CoreAddress.from("id1");
+            const sender = CoreAddress.from("Sender");
```

Comment:
I think `const peer = CoreAddress.from("Peer");` would be more suitable here, as the `owner` of the underlying IdentityAttribute of the peer shared IdentityAttribute does not need to be the Sender of the Request. In any case, an error should be thrown for peer shared IdentityAttributes.